pipenv_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Use this script to only install the required python packages depending on the environment.
+
+if [[ ${ENV} == "test" ]]; then
+    # if we want to run tests against the image, install all the dev dependencies
+    echo "Installing packages required for testing"
+    pipenv install --dev --ignore-pipfile
```

Comment:
When is this part being used? Is this for local development?

Reply:
This installs the dev packages so that we can run tests.